src/schema/v2/artwork/artworkContextGrids/__tests__/fairContext.test.ts

```diff
@@ -179,5 +177,6 @@ describe("Show Context", () => {
         "relatedArtwork2",
       ])
     })
+    expect.assertions(13)
```

Comment:
FYI As exact counts are rarely needed, you can also do just `expect.hasAssertions()` iirc